src/utils/webrtc/models/CallParticipantModel.js

```diff
@@ -127,11 +128,29 @@ CallParticipantModel.prototype = {
 		if (data.name === 'video') {
 			this.set('videoAvailable', false)
 		} else {
+			if (this.get('audioElement')) {
+				this.get('audioElement').muted = true
+			}
 			this.set('audioAvailable', false)
 			this.set('speaking', false)
 		}
 	},
 
+	forceMute: function() {
+		if (!this._peer) {
+			return
+		}
+
+		this._webRtc.sendToAll('control', {
+			action: 'forceMute',
+			peerId: this._peer.id,
+		})
+
+		// Mute locally too, as even when sending to all the sender will not
+		// receive the message.
+		this._handleMute({ id: this._peer.id })
```

Comment:
> Do you mean for the participant that force mutes another? For the participant being muted? For other participants in the conversation? For everyone?

For the actor it toggles immediately. But for the user that gets muted it takes some time on my end until the message pops in and the icon changes.
Move this above the `sendToAll`, so the UI of the participant updates immediate, instead of after the sending to others finished.

Reply:
I still do not fully understand what you mean, but if you say that moving it above fixes it, cool, I will change that.

Reply:
I have been testing with that change but I found no difference in the behaviour. But I think that now I understand what you mean (I am not sure, though).

- Enable fake streams in Firefox
- Create a public conversation with user A
- Add user B to the conversation
- Start a call as user A
- Join the call as user B
- As user A, force mute user B
- The sound stops and the icon pressed by user A changes to muted
- After a while, the local audio icon in the UI of user B changes to muted

If that is what you meant that works as expected. The sound stops because it is being played in user A's window and it is locally muted there, but even if you do not hear it user B is still sending the audio. Until user B receives the `forceMute` message it is impossible to change its audio icon.

Reply:
:man_facepalming: